### PR TITLE
fixed bug with match match any style fixed #416

### LINKED ISSUE #416: Restyle match all/any terms
As we refactored the code, you need to change the styling:
image

### PR DESCRIPTION
**Pull request by hoseinmirian**

### CODE REVIEW

**Comment by hoseinmirian** [comment]
image

**Review by knirirr — CHANGES_REQUESTED** [review]
When arriving at a page I'd expect the "match all terms" button to be selected, but it isn't. Additionally, there's a typo on the "any" button (extra "s" on "term"). 

Screenshot 2020-08-18 at 16 27 20

**Comment by hoseinmirian** [comment]
> 
> 
> When arriving at a page I'd expect the "match all terms" button to be selected, but it isn't. Additionally, there's a typo on the "any" button (extra "s" on "term").
> 
> Screenshot 2020-08-18 at 16 27 20

fixed in new commit

**Comment by knirirr** [comment]
I'm not able to achieve the desired behaviour when when pulling the latest commits from Hossein-Restyle-Match-MatchAny-#416. The button is activated on reloading a page but not when navigating to it directly via the URL or by clicking on a registry type. Are you seeing the same thing?

**Comment by hoseinmirian** [comment]
> 
> 
> I'm not able to achieve the desired behaviour when when pulling the latest commits from Hossein-Restyle-Match-MatchAny-#416. The button is activated on reloading a page but not when navigating to it directly via the URL or by clicking on a registry type. Are you seeing the same thing?

its been resolved by new commit. plz check

**Review by knirirr — APPROVED** [review]

**Review by ramGranell — APPROVED** [review]
It seems to work correctly!

File: src/store/searchFilters.js [hunk 1]
```diff
@@ -15,13 +15,13 @@ export const mutations = {
     setFilterButtons(state) {
         state.filterButtons.push([
             {
-                "title": "Match all terms",
+                "title": "Match all term",
```
Comment (terazus) [inline src/store/searchFilters.js]: If im right, the correct way is:
- match all terms (plural)
- match any term (singular)

**Review by terazus — CHANGES_REQUESTED** [review]

File: src/store/searchFilters.js (hunk 1 above)
Comment (hoseinmirian) [reply src/store/searchFilters.js]: @terazus has asked for the correctness of the spelling and as I asked @knirirr about it, I appreciate letting us know about the correct term.

Comment (terazus) [reply src/store/searchFilters.js]: I asked @knirirr and he said my proposal is right.

Comment (hoseinmirian) [reply src/store/searchFilters.js]: fixed

File: src/components/Records/Search/Input/FilterButton.vue [hunk 2]
```diff
@@ -53,13 +54,13 @@
...
             const title = _module.item.title.toLowerCase();
-            _module.checkCurrentParameters(title, fieldValue, currentValue);
+              _module.checkCurrentParameters(title, fieldValue, currentValue);
           });
         },
         methods: {
             checkCurrentParameters: function(title, fieldValue, currentValue) {
               if (fieldValue === null) {
-                this.item.active = title === 'all';
+                this.item.active = title === 'all' || title === 'match all term';
```
Comment (terazus) [inline src/components/Records/Search/Input/FilterButton.vue]: this line doesn't make sense. It's supposed to be || title === "match all terms"

**Review by terazus — CHANGES_REQUESTED** [review]

**Review by terazus — APPROVED** [review]

**Pull request closed without merging**